scripts/xsslint/xsslint/linters.py

```diff
@@ -856,13 +883,30 @@ class MakoTemplateLinter(BaseLinter):
     """
     LINE_COMMENT_DELIM = "##"
 
+
+    ruleset = RuleSet(
+        mako_missing_default='mako-missing-default',
+        mako_multiple_page_tags='mako-multiple-page-tags',
+        mako_unparseable_expression='mako-unparseable-expression',
+        mako_unwanted_html_filter='mako-unwanted-html-filter',
+        mako_invalid_html_filter='mako-invalid-html-filter',
+        mako_invalid_js_filter='mako-invalid-js-filter',
+        mako_js_missing_quotes='mako-js-missing-quotes',
+        mako_js_html_string='mako-js-html-string',
+        mako_html_entities='mako-html-entities',
+        mako_unknown_context='mako-unknown-context',
+        python_wrap_html='python-wrap-html',
```

Comment:
Having a single `python_` rule defined here has a bad smell.  Don't we get this with the visitors.ruleset?

Reply:
The `python_` rule is used directly by the `MakoLinter`, which is why it'd defined here. I didn't want to rely on it being pulled in from `visitors.ruleset` in case that ruleset changes.